src/Common/src/System/Security/Cryptography/RSASecurityTransforms.cs

```diff
@@ -53,8 +53,8 @@ public override KeySizes[] LegalKeySizes
                     {
                         // All values are in bits.
                         // 1024 was achieved via experimentation.
-                        // 1024 and 1024+64 both generated successfully, 1024-64 produced errSecParam.
-                        new KeySizes(minSize: 1024, maxSize: 16384, skipSize: 64),
+                        // 1024 and 1024+8 both generated successfully, 1024-8 produced errSecParam.
+                        new KeySizes(minSize: 1024, maxSize: 16384, skipSize: 8),
```

Comment:
Yeah, "Security Transforms" is the subset of Security.framework this is interacting with.